punch/file_configuration.py

```diff
@@ -12,7 +13,7 @@ def __init__(self, filepath, local_variables, global_variables=None):
         env = jinja2.Environment(undefined=jinja2.DebugUndefined)
         for key, value in local_variables.items():
             if six.PY2:
-                value = value.decode('utf8')
+                value = value.decode('utf8')  # pragma: no cover
```

Comment:
👎  `no cover` only helps if you want 100% coverage as a vanity metric; better that it's honestly lower than papered over.

Reply:
Well spotted! :see_no_evil: these were there temporarily just to help me check what was not covered. Then I obviously committed them. I definitely agree, will remove them.

Reply:
The report from `coverage html` is really nice - you can click around the source code and it highlights uncovered lines, as well as uncovered branches if you have branch coverage enabled. That might help next time

Reply:
Thanks, I never used it